atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamTableDefinitionBuilder.java

```diff
@@ -98,13 +99,13 @@ public TableDefinition build() {
             return new TableDefinition() {{
                 javaTableName(streamTableType.getJavaClassName(prefix));
                 rowName();
-                if (hashFirstRowComponent) {
-                    hashFirstRowComponent();
-                }
-                rowComponent("id",            idType);
+                    if (numberOfComponentsHashed > 0) {
```

Comment:
Why do you need this check?

Reply:
This was a mistake, thanks for spotting.